Netconf-notif-v12-examples-v17Sept2018/ietf-yang-push.yang

```diff
@@ -353,7 +353,7 @@ module ietf-yang-push {
        conditions that can be changed during the lifetime of the
        subscription.";
     choice update-trigger { 
-	  mandatory true;
+	  //mandatory true;

```

Comment:
Leaving this as `mandatory true` led to `yanglint` complaining that a mandtory choice was not present.